web/tests/typing_status.test.js

```diff
@@ -10,6 +10,9 @@ const compose_pm_pill = mock_esm("../src/compose_pm_pill");
 const typing = zrequire("typing");
 const typing_status = zrequire("../shared/src/typing_status");
 
+const TYPING_STARTED_WAIT_PERIOD = 10000;
+const TYPING_STOPPED_WAIT_PERIOD = 5000;
```

Comment:
Instead of setting these, I think we should be mocking/setting the page params for these values.

Reply:
I want to confirm what you mean by this.
I saw something similar in [activity.test.js ](https://github.com/Samadeolu7/zulip/blob/ccfc81290c7f6a6cf397e06a15e252ffb268c88e/web/tests/activity.test.js#L581)
Do you want me to import the empty page params and set the value for the constants as it was done there

Reply:
Okay. We're testing `typing_status.ts` here vs `typing.js`, and the constants are set in `typing.js`. So, it's probably fine as you have it.

Reply:
OK thank you very much for the review.

I will make the changes and ping you when it is ready for review.